build/vsts_build.yaml

```diff
@@ -853,6 +853,18 @@ phases:
   #       $(System.DefaultWorkingDirectory)\\VSCollect.exe -zip:$(EndToEndResultsDropPath)\\apex-collectlogs.zip
   #   condition: "failed()"
 
+  - task: NuGetCommand@2
+    displayName: "Add  .NET Core 3.0 SDK Feed Source"
+    inputs:
+      command: "custom"
+      arguments: "sources add -Name dotnet-core -Source https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json -UserName $(ApexPackageFeedUsername) -Password $(ApexPackageFeedPassword) -ConfigFile $(System.DefaultWorkingDirectory)\\NuGet.config"
```

Comment:
Why are you setting a password here? All these feeds are open. 

Also, why do we need them for the apex tests only?

Reply:
I see it's dead code. Disregard this :)